wallet-new/integration/Functions.hs

```diff
@@ -460,8 +455,10 @@ runAction wc action = do
             -- From which source to pay.
             accountSource <- pickRandomElement localAccsWithMoney
 
+            let but xs x = filter (/= x) xs
+            accountDestination <- pickRandomElement (localAccounts `but` accountSource)
```

Comment:
Isn't this virtually the same as `Data.List.(\\)`?

Reply:
Provided all accounts in the list are unique, yes (`\\` removes only the first occurrence). Will replace.

Reply:
Ah, I have to admit I missed that subtle detail -- probably yours is safer, but in theory here it shouldn't matter as account (and their indexes) are definitely unique per-wallet 😉